app/controllers/players/index.js

```diff
@@ -4,55 +4,125 @@ import { tracked } from '@glimmer/tracking';
 
 const ASCENDING_DIRECTION = 'ASC';
 const DESCENDING_DIRECTION = 'DESC';
+const DOWN_ARROW = '&darr;';
+const UP_ARROW = '&uarr;';
 
 export default class IndexController extends Controller {
-    get pageArray() {
-        let array = []
-
-        for (let index = 1; index <= Math.ceil(this.model.meta.count / this.limit); index++) {
-            array.push(index);
-        }
+    get pageValues() {
+        let array = new Array(Math.ceil(this.model.meta.count / this.limit)).fill().map((_, i) => i + 1);
```

Comment:
1. Can we define this variable as `const`?
2. How about using a better name for this variable? Something like `pages`?
3. Is it possible to move the `Math.ceil(this.model.meta.count / this.limit)` logic to separate variable?

Reply:
`Array.from({length: Math.ceil(this.model.meta.count / this.limit)}, (_, i) => i + 1)`